djangocms_navigation/models.py

```diff
@@ -28,8 +28,7 @@ class Meta:
 
 class MenuContent(models.Model):
     menu = models.ForeignKey(Menu, on_delete=models.PROTECT)
-    root = models.ForeignKey(
-        'djangocms_navigation.MenuItem', on_delete=models.PROTECT)
+    root = models.ForeignKey("djangocms_navigation.MenuItem", on_delete=models.PROTECT)
```

Comment:
Technically speaking you've changed it from pep8 compliant to non-pep8 compliant - just sayin' ;) I know max line length isn't a big deal and there are some cases where to make the line short is to make things very difficult to read, but sometimes when doing merges in split screen etc. it's actually useful to have the lines under 80 chars.

Reply:
I use black to format code and max line length is taken from setup.cfg which is currently 120 for all our projects.